Resculpt/Resculptable.cs

```diff
@@ -90,6 +90,22 @@ public sealed class Resculptable : KMonoBehaviour {
 			}
 		}
 
+
+		/// <summary>
+		/// Triggered when the user requests a rotation of the decor item.
+		/// </summary>
+		private void OnRotateClicked()
+		{
+			Rotatable rotatable = this.gameObject.GetComponent<Rotatable>();
+			if (rotatable != null) rotatable.Rotate();
+
+			// Buildings with even width values jump one tile when rotating and must be moved back
+			BuildingDef def = this.gameObject.GetComponent<Building>()?.Def;
+			if (def != null && def.WidthInCells % 2 == 0)
+				this.transform.position += rotatable.GetOrientation() != Orientation.Neutral ? new UnityEngine.Vector3(1, 0, 0)
```

Comment:
This statement could throw if `rotatable == null`, perhaps this should be included in the `if (rotatable != null)` block

Reply:
included in if block